Module.java

```diff
@@ -0,0 +1,38 @@
+package extraction.sProject;
+
+public class Module {
+	
+	private String moduleID;
+	private double moduleScore;
+	
+	public Module() {
+		super();
+		this.moduleID = "N/A";
+		this.moduleScore = 0;
```

Comment:
Same as above, `-1` may be a better default. Except here there should never be a score below 0.